default/python/AI/PriorityAI.py

```diff
@@ -228,7 +228,10 @@ def _calculate_colonisation_priority():
     if num_colonies > colony_growth_barrier:
         return 0.0
     colony_cost = AIDependencies.COLONY_POD_COST * (1 + AIDependencies.COLONY_POD_UPKEEP * num_colonies)
-    turns_to_build = 8  # TODO: check for susp anim pods, build time 10
+    if tech_is_complete(AIDependencies.GRO_LIFE_CYCLE):
```

Comment:
That looks super confusing.  Lest improve it by introducing constants. 

```python
AIDependencies.COLONY_POD_TURNS_TO_BUILD = 8
AIDependencies.CRYONIC_COLONY_POD_TURNS_TO_BUILD = 10  
AIDependencies.CRYONIC_COLONY_POD_COST = 120  
```

And moving logic to the function.

```python
def get_colony_pod_stats(num_colonies: int) -> [int, float]:
    """
    Calculate turns to produce and cost  for colony pod.

    When more than one pod type is available we just choose most advanced. 
    """
    if tech_is_complete(AIDependencies.GRO_LIFE_CYCLE):
        turn_to_complete = AIDependencies.CRYONIC_COLONY_POD_TURNS_TO_BUILD
        base_cost = AIDependencies.CRYONIC_COLONY_POD_COST
    else:
        turn_to_complete = AIDependencies.COLONY_POD_TURNS_TO_BUILD
        base_cost = AIDependencies.COLONY_POD_COST
    return turn_to_complete, base_cost * (
                1 + AIDependencies.COLONY_POD_UPKEEP * num_colonies)
```

Reply:
Can those constants be acquired from freeorion instance?

Reply:
Good point.   Yes, it can be done, and it should be done.  

Can we check if part available?   In that case, we can skip the tech check and just iterate over pads and choose first suitable.

Reply:
There is `availableShipParts` property:

https://github.com/freeorion/freeorion/blob/4fc3ad995b3845bbe4ddfa4618fad1781eedc267/python/EmpireWrapper.cpp#L323